### DISCUSSION TITLE
Remote Georgia Tech Student Rejected Student Developer Pack

### DISCUSSION THREAD

**Discussion by younghk** [community/community#56647] [GitHub Education]
Hi,  I recently applied for the GitHub Student Developer Pack but received an email indicating that my application was denied. The reasons provided were:

- The school I selected does not appear to have a campus location in my country and my school-provided academic affiliation document must indicate virtual learning if I am remote.
- I should consider using my device camera to submit academic affiliation documents as uploaded images are less trustworthy.
- I appear not to be near any campus location for the school I have selected, and if I am a distance learner then my school-provided academic affiliation documentation must state so.

To address these issues, I would like to clarify that I am currently enrolled in a master's degree program at the mentioned institution and am studying remotely. When I submitted my application, I included a proof document that states my degree candidate status, obtained from my school's online site. However, it appears that this document may not have clearly indicated my status as a remote student.

In light of the feedback you provided, I am willing to take a photo using my device camera of additional academic documentation, showing that I am indeed a remote learner, if that would be more acceptable. If there is a specific type of document or format that you require for this, could you please let me know?

Also, if there are any other details or additional information required to increase the chances of my application being accepted, I would greatly appreciate your guidance.

**Comment by jge162** [comment]
You need to use your school associated email (Georgia tech(@) as your main email on your GitHub account number 1. Then include a document showing your current enrollment. Either the transcripts with your name and school name. Or a school Id. Good luck.

**Comment by younghk** [comment]
@jge162 thank you for quick reply! 
I set my email and retried but I still got some errors. 
The email that I got back is attached as below.

<img width="606" alt="image" src="https://github.com/community/community/assets/25054007/c38ca456-1e00-48fa-a382-c68a70f2cc40">

> You need to use your school associated email (Georgia tech(@) as your main email on your GitHub account number 1

Do I need to sign-up with my school email? Originally I use my personal email and add school email later after I started my degree.
When I check my account setting, it looks like

<img width="780" alt="스크린샷 2023-05-29 10 42 46" src="https://github.com/community/community/assets/25054007/fecef1dc-8afe-493e-ba3a-d4073ac7cbce">

Is there any other way to resolve this error or do I miss something else ?

**Reply by jge162** [reply]
You need a document probably that shows you are remote. Because, your ip address might show you are nowhere near the campus I guess.

**Reply by alpbalcay** [reply]
Is this resolved?
